modules/nw-sriov-supported-devices.adoc

```diff
@@ -39,7 +39,17 @@
 
 |Intel
 |E810-CQDA2
-|8088
+|8086
+|1592
+
+|Intel
+|E810-2CQDA2
+|8086
+|1592
+
+|Intel
```

Comment:
E810-XXVDA4 already exist in the following, no need to add a new one,  we only need to update the 8088 to 8086

Reply:
/lgtm, thanks